mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/ProfileItemCard.kt

```diff
@@ -0,0 +1,82 @@
+package org.mifos.mobilewallet.mifospay.home.ui
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.mifos.mobilewallet.mifospay.ui.utility.Orientation
+
+@Composable
+fun ProfileItemCard(
```

Comment:
Explore the Flow https://developer.android.com/jetpack/compose/layouts/flow#item-weights this Horizontal and Vertical thing can be handled directly through the single below Flow. See example in above Flow documentation.

Reply:
Wont the FlowRow cause inconsistent UI for different screen sizes? In case of screens with smaller width the elements will flow out.

Reply:
No, Flow is built for that specially.